modules/firewall-rules/versions.tf

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = ">=0.12.6, <0.14"
```

Comment:
is this is a breaking change as version 3.0 didn't require <0.14 and now 3.1 does ?

Reply:
@morgante

Reply:
and i just saw your pr for the fix (https://github.com/terraform-google-modules/terraform-google-network/pull/245)  ignore me.